Lib/test/test_builtin.py

```diff
@@ -75,6 +75,23 @@ class BitBucket:
     def write(self, line):
         pass
 
+class MalformedInputStream:
```

Comment:
"Malformed" in what way? I think this name could be better.
```suggestion
class InputStreamWithReadlineButNoRead:
```